src/main/java/org/folio/pv/client/HashedPasswordUsageCollectionConverter.java

```diff
@@ -140,7 +143,10 @@ private T createCollection(Class<?> collectionClass) {
       try {
         return (T) ReflectionUtils.accessibleConstructor(collectionClass).newInstance();
       } catch (Exception ex) {
-        throw new IllegalArgumentException("Could not instantiate collection class: " + collectionClass.getName(), ex);
+        IllegalArgumentException e =
+          new IllegalArgumentException("Could not instantiate collection class: " + collectionClass.getName(), ex);
+        log.warn(e.getMessage());
+        throw e;
```

Comment:
maybe it's better to name it `exception`

Reply:
It looks good when it's in on line
![image](https://user-images.githubusercontent.com/88006020/226637301-19946ca2-467b-4ef3-aa77-cb1bb24ba2e9.png)